dashboard/src/deployment/DeploymentList.js

```diff
@@ -0,0 +1,191 @@
+        {(license==="enterprise") ? <Popup trigger={<Icon name="dollar"/>} content="Enterprise edition"/>: null}
+      </span>
+    </Table.Cell>
+    <Table.Cell>{readyPodCount} / {podCount}</Table.Cell>
+    <Table.Cell>{readyVolumeCount} / {volumeCount}</Table.Cell>
+    <Table.Cell>{storageClasses.map((item) => (item === "") ? "<default>" : item)}</Table.Cell>
+    <Table.Cell>
+      { databaseURL ? <DatabaseLinkView name={name} url={databaseURL}/> : <NoDatabaseLinkView/>}
+      <CommandInstruction 
+          trigger={<Icon link name="zoom"/>}
+          command={describeCommand}
+          title="Describe deployment"
+          description="To get more information on the state of this deployment, run:"
+        />
+      <span style={{"float":"right"}}>
+        <CommandInstruction 
+          trigger={<Icon link name="trash"/>}
+          command={deleteCommand}
+          title="Delete deployment"
+          description="To delete this deployment, run:"
+        />
+      </span>
+    </Table.Cell>
+  </Table.Row>
+);
+
+const ListView = ({items, loading}) => (
+  <Table striped celled>
+    <HeaderView loading={loading}/>
+    <Table.Body>
+      {
+        (items) ? items.map((item) => 
+          <RowView 
+            key={item.name} 
+            name={item.name}
+            namespace={item.namespace}
+            mode={item.mode}
+            environment={item.environment}
+            stateColor={item.state_color}
+            version={item.database_version}
+            license={item.database_license}
+            readyPodCount={item.ready_pod_count}
+            podCount={item.pod_count}
+            readyVolumeCount={item.ready_volume_count}
+            volumeCount={item.volume_count}
+            storageClasses={item.storage_classes}
+            databaseURL={item.database_url}
+            deleteCommand={createDeleteCommand(item.name, item.namespace)}
+            describeCommand={createDescribeCommand(item.name, item.namespace)}
+          />) : <p>No items</p>
+      }
+    </Table.Body>
+  </Table>
+);
+
+const EmptyView = () => (<div>No deployments</div>);
+
+function createDeleteCommand(name, namespace) {
+  return `kubectl delete ArangoDeployment -n ${namespace} ${name}`;
+}
+
+function createDescribeCommand(name, namespace) {
+  return `kubectl describe ArangoDeployment -n ${namespace} ${name}`;
+}
+
+function getStateColorDescription(stateColor) {
+  switch (stateColor) {
+    case "green":
+      return "Everything is running smooth.";
+    case "yellow":
+      return "There is some activity going on, but deployment is available.";
+    case "orange":
+      return "There is some activity going on, deployment may be/become unavailable. You should pay attention now!";
+    case "red":
+      return "The deployment is in a bad state and manual intervention is likely needed.";
+    default:
+      return "State is not known.";
+  }
+}
+
+class DeploymentList extends Component {
+  state = {
+    items: undefined,
```

Comment:
If you need an explicit "this is a thing but it's not set yet" value, `null` is generally preferable to `undefined` as it's difficult to distinguish values that exist but are `undefined` form values that are undefined because they don't exist.

Reply:
fixed